kornia/testing/__init__.py

```diff
@@ -205,9 +205,9 @@ def KORNIA_CHECK_SHAPE(x, shape: List[str]) -> None:
             raise TypeError(f"{x} shape should be must be [{shape}]. Got {x.shape}")
 
 
-def KORNIA_CHECK(condition: bool, msg: Optional[str] = None):
+def KORNIA_CHECK(condition: bool, msg: Optional[str] = None, exe_cls=Exception):
     if not condition:
-        raise Exception(f"{condition} not true.\n{msg}")
+        raise exe_cls(f"{condition} not true.\n{msg}")
```

Comment:
Maybe do this in a separated pr to not block this one

Reply:
yes I removed it :(